packages/color-area/src/ColorArea.ts

```diff
@@ -72,18 +73,7 @@ export class ColorArea extends SpectrumElement {
 
     private languageResolver = new LanguageResolutionController(this);
 
-    private colorController = new ColorController(this, {
-        extractColorFromState: () => ({
-            h: this.hue,
-            s: this.x,
-            v: this.y,
-        }),
-        applyColorToState: ({ s, v }) => {
-            this._x = s;
-            this._y = v;
-            this.requestUpdate();
-        },
-    });
+    private colorController = new ColorController(this, { manageAs: 'hsv' });
```

Comment:
I have added docs block in color-controller and explained the use of manage as there.
Also this manageAs parameter is not required for/by colorjs.io . It already existed before migrating to colorjs when we were using ctrl/tinycolor.

`manageAs` defines the color space in which the color object will be managed. It is used to precisely perform the calculations whenever the sp-color component is altered. For instance in an sp-color-area the `hue` value is constant thus when we drag the handle only the `saturation` and `value` cords change. `saturation` varies along x axis of color area and `value` varies along `y` axis of color area. Thus these `s` and `v` values are mapped to the x and y axis of color area. Whenever user changes the value of color area along x axis by say 1 unit the value of color should also change accordingly and since x axis corresponds to saturation value so the s value should also change accordingly. Now lets imagine a case where we don't leverage `manageAs` property and the colorController is using `srgb` color space. When the user drags the color handle by 1 unit the color object is first converted to `hsv` color space(which is although very little b ut lossy) then the `s` value is changed by 1 unit then again converted to `srgb` color space(again although very little but lossy). But if we are using manageAs and passing its value as "hsv" the colorController knows that for this host the color value will be managed(internally) in `hsv` color space. Now if the user drags the handle on color area by 1 unit the colorController which is managing the color value in `hsv` color space will just alter the value of `s` saturation by 1 unit and viola!, no loss happens.